src/Microsoft.OData.Edm/ExtensionMethods/EnumHelper.cs

```diff
@@ -126,6 +126,48 @@ public static bool TryParseEnum(this IEdmEnumType enumType, string value, bool i
             return true;
         }
 
+        /// <summary>

```

Comment:
Do the two methods need to be public? Probably not. Make them `internal` if possible and remove the public API changes.

Reply:
@gathogojr 

The two methods are in `Microsoft.OData.Edm/ExtensionMethods/EnumHelper.cs` and I want to use them in a different project `Microsoft.OData.Core/UriParser/Binders/MetadataBindingUtils.cs`. If I make them `internal`, they are accessible in `Microsoft.OData.Core` project. If I make them `public`, they must be added to Public API.

Reply:
@gathogojr 

I have moved the 2 methods to [src/Microsoft.OData.Core/EdmExtensionMethods.cs](https://github.com/OData/odata.net/pull/3014/files#diff-9515e6bf05155cabcb114b3a0b071152eae0435663121151fde82b3fccffdb0b)